modules/gitops-release-notes-1-10-0.adoc

```diff
@@ -0,0 +1,120 @@
+----
+
+[id="new-features-1-10-0_{context}"]
+== New features
+
+The current release adds the following improvements:
+
+* With this update, the Argo CD CRD API version is upgraded from `v1alpha1` to `v1beta1` in order to accomodate the breaking changes resulting from the deprecation of `.spec.dex` and certain `.spec.sso` fields. To streamline the automatic migration of existing `v1alpha1` Argo CD CRs to `v1beta1`, conversion webhook support has been implemented. link:https://issues.redhat.com/browse/GITOPS-3040[GITOPS-3040]
++
+[NOTE]
+====
+By default, the conversion webhook is enabled only for OLM-installed Operators. For non-OLM installations of the Operator, enabling the webhook is optional. However, without conversion webhook support, you have to manually migrate any existing Argo CD `v1alpha1` CRs to `v1beta1`.
+====
+
+* With this update, the {gitops-title} Operator deploys three monitoring dashboards in the Administrator perspective of the web console. The three dashboards are: *GitOps Overview*, *GitOps Components*, and *GitOps gRPC*. To access these dashboards, go to *Observe* → *Monitoring*. link:https:https://issues.redhat.com/browse/GITOPS-1767[GITOPS-1767]
++
+[NOTE]
+====
+Disabling or changing the content of the dashboards is not supported.
+====
+
+* With this update, the {gitops-title} Operator now gathers and exports metrics data that can be handled by Prometheus. The metrics data collected by the server include the following:
++
+** The total number of Argo CD instances managed and reconciled.
+** The number of Argo CD instances in a given state, such as pending, available, among others. 
+** The total time for the reconciliation loops to complete per instance,
+** The amount of resources used such as CPU, memory, and number of goroutines.
++
+To access these metrics, go to the *Observe* tab on the web console, and run queries against the monitoring stack. link:https://issues.redhat.com/browse/GITOPS-2645[GITOPS-2645]
++
+[NOTE]
+====
+You need to install the {gitops-title} Operator in the default `openshift-gitops-operator` namespace, in order to have these metrics automatically available.
+====
+
+* Previously, timestamps were presented in a Unix epoch format. With this update, the timestamps are changed to RFC3339 format, for example: 2023-06-27T07:12:48-04:00, to improve overall readability. link:https://issues.redhat.com/browse/GITOPS-2898[GITOPS-2898]
+
+* With this update, the default Argo CD instance in the `openshift-gitops` namespace has restricted permissions for non-admin users by default. This improves security because non-admin users no longer have access to sensitive information. However, as an administrator, you can set permissions and grant non-admin users access to the resources managed by the default `openshift-gitops` Argo CD instance by configuring your Argo CD RBAC. This change only applies to the default {gitops-title} instance. link:https://issues.redhat.com/browse/GITOPS-3032[GITOPS-3032]
```

Comment:
As I said on the jira release notes text, I think we should add a link to the docs  for configuring Argo CD RBAC if possible (https://docs.openshift.com/container-platform/4.12/cicd/gitops/configuring-argo-cd-rbac.html)?

Reply:
Noted

Reply:
@reginapizza let us refrain from adding doc links here as we are going standalone. This will add overload in changing redirect links for us due to the changes in file location.

Reply:
Noted!